client/gutenberg/extensions/related-posts/index.js

```diff
@@ -0,0 +1,52 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import edit from './edit';
+import { MAX_POSTS_TO_SHOW } from './constants';
+
+registerBlockType( 'a8c/related-posts', {
+	title: __( 'Related Posts' ),
+
+	icon: 'admin-page',
+
+	category: 'layout',
+
+	attributes: {
+		postLayout: {
+			type: 'string',
+			default: 'grid',
+		},
+		headline: {
+			type: 'string',
+			default: __( 'Related' ),
+		},
+		displayDate: {
+			type: 'boolean',
+			default: true,
+		},
+		displayThumbnails: {
+			type: 'boolean',
+			default: false,
+		},
+		displayContext: {
+			type: 'boolean',
+			default: true,
+		},
+		postsToShow: {
+			type: 'number',
+			default: MAX_POSTS_TO_SHOW,
+		},
+	},
+
+	edit,
+	save: () => null,
```

Comment:
is there a problem in providing the view output from the editor as the fallback render? it seems like it'd be nice to have a static render since it's available. we could split out the bit from `edit` that returns the HTML and then reuse it here

Reply:
Also, the latest posts core block, which has a kind of similar purpose, seems to be following [the same approach](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/latest-posts/index.js#L38).